apps/openmw/mwworld/projectilemanager.cpp

```diff
@@ -92,7 +94,18 @@ namespace MWWorld
             attachTo = rotateNode;
         }
 
-        mResourceSystem->getSceneManager()->getInstance(model, attachTo);
+        osg::ref_ptr<osg::Node> ptr = mResourceSystem->getSceneManager()->getInstance(model, attachTo);
+
+        if (state.mIdMagic.size() > 1)
+            for (size_t iter = 1; iter != state.mIdMagic.size(); ++iter)
+            {
+                char numstr[8];
+                sprintf(numstr, "%d", (int)iter);
+                std::string node = "Dummy0";
```

Comment:
Oops, you are right.

Then use the C++ way:

<pre>
std::ostringstream nodeName;
nodeName << "Dummy0" << iter;
std::string nodeNameStr = nodeName.str();
</pre>